screen-pop-app/screen-pop/src/ConversationManager.tsx

```diff
@@ -0,0 +1,53 @@
+
+export interface ConfigData {
+    lastState: string; 
+    expiry: number; 
+}
+
+interface Manager {
+    [key: string]: ConfigData;
+}
+
+export class ConversationManager {
+
+
+static conversationKey = "conversation"
+
+static saveItem(interactionID : string, lastState: string, ttl: number = (1000 * 60 * 60 * 24) ) : void {
+  const now = new Date() 
+  const value = {
+    lastState: lastState, 
+    expiry: now.getTime() + ttl
+  }
+  var rawValue = localStorage.getItem(this.conversationKey)
+  if(!rawValue){ // no data has been saved previously 
+    const data: Manager = {interactionID : value}   
+    localStorage.setItem(this.conversationKey, JSON.stringify(data))
+    return
+  }
+var dataBank : Manager = JSON.parse(rawValue);
+  
+  dataBank[interactionID] = value; 
+  localStorage.setItem(this.conversationKey, JSON.stringify(dataBank))
+}
+
+static getItem(interactionID: string) : null | ConfigData{
+    var rawValue = localStorage.getItem(this.conversationKey)
```

Comment:
Be sure you're using prettier and formatting your files. Indentation and other formatting is inconsistent.